content-tree.d.ts

```diff
@@ -753,5 +753,17 @@ export declare namespace ContentTree {
             children: [TableCaption, TableBody, TableFooter];
             columnSettings: TableColumnSettings[];
         }
+        interface CustomCodeComponent extends Parent {
+          type: 'custom-code-component';
+          path: string;
+          versionRange: string;
+          lastModified: string;
```

Comment:
What does `lastModified` refer to here? Would a component ever have a different `lastModified` to the published date of an article? 🤔 It doesn't appear to be used in cp-content-pipeline by anything...

Reply:
Thanks @adgad 
This property is currently an [upcoming feature](https://financialtimes.atlassian.net/browse/SP-3708) which will be implemented before the end of the next sprint.
I can remove for now, and re-add at a later date, if you feel this is required

Reply:
hey @adgad - we've changed the name to `attributesLastModified` - it's the time at which the attributes were last changed. We haven't implemented the feature yet but thought it might make sense to include it in this PR 👍